code/game/objects/items/devices/megaphone.dm

```diff
@@ -3,7 +3,7 @@
 	desc = "A device used to project your voice. Loudly."
 	icon_state = "megaphone"
 	item_state = "radio"
-	w_class = 1
+	w_class = WEIGHT_CLASS_SMALL
```

Comment:
Actually to simplify it, I'm going to remove it from this PR entirely and just make a separate one for it... Atomic commits and all that...